streaming/rrdpush.c

```diff
@@ -40,6 +40,9 @@ struct config stream_config = {
 };
 
 unsigned int default_rrdpush_enabled = 0;
+#ifdef ENABLE_COMPRESSION
+unsigned int default_compression_enabled = 0;
```

Comment:
Why are you adding compression as disabled by default? I think this is a very positive feature, and instead to be disabled, it should be enabled by default.

Reply:
Hello thiagoftsm,
I didn't enabled stream data compression by default because there is limitation (16K) for maximal block size.

Reply:
All right, I understood now, thanks!